packages/config/config/devices/0x031e/lzw60.json

```diff
@@ -65,17 +75,18 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Disabled",
+					"label": "No",
```

Comment:
There is a specific comment about avoiding yes/no, converting to enable/disable, and using the approbation base.

Reply:
Sorry, I apparently amended the commit and pushed, but had forgotten to add the file I was committing with, should be fixed